tests/test_pipeline_mixin.py

```diff
@@ -179,11 +181,12 @@ def run_model_pipeline_tests(self, task, repo_name, model_architecture, tokenize
                     tokenizer_name,
                     processor_name,
                 ):
-                    self.skipTest(
+                    logger.warning(
```

Comment:
We should NOT skip at this moment - as we might skip the next combination of tokenizer/processor.